tests/bsim/bluetooth/host/l2cap/many_conns/src/main.c

```diff
@@ -0,0 +1,488 @@
+/* Application main entry point */
+
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bstests.h"
+#include "common.h"
+
+#define LOG_MODULE_NAME main
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL_INF);
+
+CREATE_FLAG(is_connected);
+CREATE_FLAG(flag_l2cap_connected);
+
+#define __stronk
```

Comment:
... :D


```suggestion
```

Reply:
on it :+1: 

![download (2)](https://github.com/zephyrproject-rtos/zephyr/assets/68428270/db9996b7-6378-4079-aaeb-05d1f788e90e)